internal/pkg/cli/command/target/target.go

```diff
@@ -52,7 +38,15 @@ func NewTargetCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "target <flags>",
 		Short: "Set context for the CLI",
-		Long:  targetHelp,
+		Long: help.Long(`
```

Comment:
Does the target command give an error if you try to use it while authing with API key?